### ISSUE TITLE
Hook into FWP

### ISSUE THREAD

**Issue by cogdog** [vcualtlab/list-remote-comments#2]
I've never looked to closely at the Feed Wordpress code, but I understand there are hooks into it (other plugins that work off of it like Advanced Filters); might be possible to hook in the checks for remote comments into the cycle of FWP updates; that way you would not need to do an update check (one less post-meta too). 

Then again, that might mean multiple remote comment feed requests....

The thing is, as written, that the updates would only be triggered upon a single view of the post (?)

**Comment by luetkemj** [comment]
I haven't looked too closely at feedwordpress yet either. As I mentioned in [issue1](https://github.com/vcualtlab/list-remote-comments/issues/1) this was all based on the remote comments plugin. 

Some great places to dig further.

**Comment by luetkemj** [comment]
> The thing is, as written, that the updates would only be triggered upon a single view of the post (?)

Not sure I follow, this plugin should work on single posts or the main loop - but with feedwordpress - at least how we are implementing things, you wouldn't ever see the single view as it direct links you to the child blog.

**Comment by cogdog** [comment]
Yeah, that was not fully thought out.

This was the experience using the code Martha originally wrote, it was
executed for every item in the loop, on every pass (there was no caching).
So running on an archive view of posts, the script was waiting on a remote
RSS fetch for maybe 10 or 20 sites, and it was a bottle neck (and then it
was just to get a comment count).

So how you space things out for the remote calls matters; in your case, and
it seems reasonable, you do not even look more than once an hour. If a host
is slow to respond, or does not, the loop might stall.

I might look at a way, perhaps to offload the looking for comments to
something on a cron script? But if you are storing the comment or its
title, you have to store that somewhere.

The other way, what we built last year for thought vectors, was a second
comment reader site that just FWPed comments form blogs that were in the
main site, so it becomes a data store of comments.

This of course would not be something a faculty could do easily for one
site. Maybe it's not intensive to check as needed for comments at the
single post view.

I've not looked at the remote comments plugin yet, so I will stop wild
guesses. But where/how the remote calls are triggered is pretty important.

On Tue, Jun 9, 2015 at 1:44 PM, Mark Luetke notifications@github.com
wrote:

> The thing is, as written, that the updates would only be triggered upon a
> single view of the post (?)
> 
> Not sure I follow, this plugin should work on single posts or the main
> loop - but with feedwordpress - at least how we are implementing things,
> you wouldn't ever see the single view as it direct links you to the child
> blog.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/vcualtlab/list-remote-comments/issues/2#issuecomment-110498994
> .

## 

_--------------------------------------------------------------------Alan
Levine: A Cog, A Dog, A Blog_
cogdogblog.com • cogdog.info
StoryBox • cogdogblog.com/storybox

photoing on flickr http://flickr.com/photos/cogdog & Barking Dog Studios
http://barkingdog.me • tweeting @cogdog http://twitter.com/cogdog

## 50+ Web 2.0 Ways to Tell a Story -- 50ways.wikispaces.com

**Comment by luetkemj** [comment]
> So running on an archive view of posts, the script was waiting on a remote
> RSS fetch for maybe 10 or 20 sites, and it was a bottle neck

Yes. Uncached these can take several seconds to load up. Cached it's pretty snappy. Will look at a chron or even more aggressive caching. Not sure how often these even need to be updated. They are more a tool for faculty than an active part of the course itself.
